operator/hack/common.sh

```diff
@@ -38,11 +39,19 @@ function apply_operator_manifests() {
   local -r image_tag_base="$2"
   local -r index_version="$3"
   local -r operator_version="$4"
+  OPERATOR_YAML_PATH="operator/hack/operator.envsubst.yaml"
+  # Get Operator channel from json for midstream
+  if [[ "${USE_MIDSTREAM_IMAGES}" == "true" ]]; then
+    operator_channel=$(< midstream/iib.json jq -r '.operator.channel')
+    OPERATOR_YAML_PATH="operator/hack/operator-midstream.envsubst.yaml"
+  fi
+
   env -i PATH="${PATH}" \
-    INDEX_VERSION="${index_version}" OPERATOR_VERSION="${operator_version}" NAMESPACE="${operator_ns}" \
+    INDEX_VERSION="${index_version}" OPERATOR_VERSION="${operator_version}" NAMESPACE="${operator_ns}" OPERATOR_CHANNEL="${operator_channel}" \
```

Comment:
operator_channel has no default value when running with non midstream images, causing failures seen in other   ocp tests @mdafsanhossain

Reply:
We can keep it separate for midstream then like it was previously.